.github/workflows/refapp-3x-login.yml

```diff
@@ -16,9 +12,16 @@ jobs:
     runs-on: ubuntu-latest
     defaults:
       run:
-        working-directory: qaframework-bdd-tests
+        working-directory: ./qaframework-bdd-tests
     steps:
-      - uses: actions/checkout@v2
+      - name: Checkout qaframework
+        uses: actions/checkout@master
+        with:
+          repository: ${{github.repository}}
+      - name: Run db and web containers
+        run: docker-compose -f docker/docker-compose-refqa-3x.yml up -d
+      - name: Wait for Openmrs instance to start
+        run: while [[ "$(curl -s -o /dev/null -w ''%{http_code}'' http://localhost/openmrs/spa/login)" != "200" ]]; do sleep 2; done
```

Comment:
Maybe spa UI is starting immediately. Check it on your local instance if it begins immediately, then we need to find another way to wait until the server starts.

Run your local instance and find a URL that won't load until the server is up. 

Check this one as well:
http://localhost:8080/openmrs/login